recipes/opennb/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "opennb" %}
+{% set version = "0.2.3" %}
+{% set python_min = "3.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7f9e20f9872ce7d7ba47e4a22d9bb1a639007c6b43a9ee8c5bcb901c7e7b62ed
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - opennb = opennb.__main__:main
+
+requirements:
+  host:
+    - python {{ python_min }}.*
+    - hatchling
+    - pip
+  run:
+    - python >={{ python_min }}
+    - notebook
+
+test:
+  imports:
+    - opennb
+  commands:
```

Comment:
Can we test the entry point with `opennb --help` under `commands`?

Reply:
Thanks for the suggestion @synapticarbors!

I have added it and the builds are passing ✅